frontend/packages/dev-console/integration-tests/features/addFlow/create-operator-backed.feature

```diff
@@ -11,14 +11,15 @@ Feature: Create workload from Operator Backed file
 # Below scenario needs to be executed only once. Second time it throws error. So not recommended to execute in automation suite
         @regression @manual
         Scenario: Create the workload from Operator Backed: A-08-TC01
-            Given user has created or selected namespace "knative-eventing"
+            Given user has not created kafka instance in "knative-eventing" namespace
+              And user has created or selected namespace "knative-eventing"
               And user is at OperatorBacked page
              When user selects knative Kafka card
               And user clicks Create button in side bar
               And user enters name as "knative-kafka" in Create knative Serving page
               And user clicks create button in Create knative Serving page
              Then user will be redirected to Topology page
-              And user is able to see workload "knative-serving-1" in topology page
+              And user is able to see workload "knative-kafka" in topology page
```

Comment:
This doesn't work I can't see "knative-kafka" in topology page

Reply:
updated